### PR TITLE
[Design] [MG] Add anticipated and unlikely changes

### LINKED ISSUE #346: [Design] [MG] Anticipated Changes
## Content

- Anticipated Changes

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #347: [Design] [MG] Unlikely Changes
## Content

- Unlikely Changes

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### PR DESCRIPTION
**Pull request by LeonSo7**
[Design][MG] Add anticipated and unlikely changes

Close https://github.com/arkinmodi/project-sayyara/issues/346
Close https://github.com/arkinmodi/project-sayyara/issues/347

### CODE REVIEW

**Comment by arkinmodi** [comment]
Just for better formatting. I prefer not to use the prof's custom commands and think that native latex is cleaner (and less lines). Also remember to give each point a label so it can be referenced in the "Trace Between Anticipated Changes and Modules" table.

**Anticipated Changes**
```latex
\begin{enumerate}[label=\textbf{AC\arabic*:},ref=AC\arabic*]
	\item \label{acHardware} The specific hardware on which the software is running.
	\item \label{acInput} The format of the initial input data.
	\item The format of the initial input data
	\item The format of server-side responses
	\item The styling of UI forms and components
	\item The formatting of UI forms and components
\end{enumerate}
```

**Unlikely Changes**
```latex
\begin{enumerate}[label=\textbf{UC\arabic*:},ref=UC\arabic*]
	\item \label{ucIO} Input/Output devices (Input: File and/or Keyboard, Output: File, Memory, and/or Screen).
	\item The application will be built-upon the NextJS framework
	\item The application will use Prisma for object-relation mapping (ORM)
	\item There will always be a source of input data external to the system
\end{enumerate}
```

With this, in the "Trace Between Anticipated Changes and Modules" table, you can just use `\ref{...}` instead of `\acref{...}` and `\mref{...}`.

File: docs/Design/SoftArchitecture/MG.tex [hunk 1]
```diff
@@ -175,7 +179,9 @@ \subsection{Unlikely Changes} \label{SecUchange}
 \begin{description}
 	\item[\refstepcounter{ucnum} \uctheucnum \label{ucIO}:] Input/Output devices
 		(Input: File and/or Keyboard, Output: File, Memory, and/or Screen).
-	\item ...
+	\item The application will be built-upon the NextJS framework
```
Comment (arkinmodi) [inline docs/Design/SoftArchitecture/MG.tex]:
```suggestion
	\item The application will be built-upon the Next.js framework
```

File: docs/Design/SoftArchitecture/MG.tex [hunk 2]
```diff
@@ -175,7 +179,9 @@ \subsection{Unlikely Changes} \label{SecUchange}
...
+	\item The application will use Prisma for object-relation mapping (ORM)
```
Comment (arkinmodi) [inline docs/Design/SoftArchitecture/MG.tex]: Also could instead just add ORM to the "Abbreviations and Acronyms" table.
```suggestion
	\item The application will use Prisma for object-relational mapping (ORM)
```

**Review by arkinmodi — COMMENTED** [review]
I think the points you have are good enough.

File: docs/Design/SoftArchitecture/MG.tex (hunk 2 above)
Comment (LeonSo7) [reply docs/Design/SoftArchitecture/MG.tex]: Added ORM to the table

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**